MeshRender/OpenGLTest/Renderer.cpp~

```diff
@@ -0,0 +1,135 @@
+#include "Renderer.h"
+extern GLFWwindow* window;
+
+
+Renderer::Renderer()
```

Comment:
http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#c80-use-default-if-you-have-to-be-explicit-about-using-the-default-semantics